system-tests/helm/README.md

```diff
@@ -0,0 +1,115 @@
+# Helm Charts for MVD
+We have demonstrated a containerized deployment of the MVD in [system-tests/README.md](../README.md). To take it one step further
+we will be using [kubernetes](https://kubernetes.io/docs/home/) which is a container orchestration tool for automated deployment, scaling, and management of these containers.
+In addition, we have used [Helm](https://helm.sh/docs/) to manage all the Kubernetes YAML files.
+[./helm-charts](./helm-charts) folder contains the helm charts for the kubernetes components of the applications used in MVD.
+
+
+## Deploying MVD
+For the deployment purpose we need,
+* a kubernetes cluster, and
+* helm installed
+
+You can use [minikube](https://minikube.sigs.k8s.io/docs/) or [kind](https://kind.sigs.k8s.io/)
```

Comment:
suggestions(non-blocking): A requirements section could be cleaner, to see which applications and version numbers have to be installed.